pallets/ajuna-nft-transfer/src/lib.rs

```diff
@@ -93,6 +104,8 @@ pub mod pallet {
 
 		type AccountManager: AccountManager<AccountId = AccountIdFor<Self>>;
 
+		type FeeHandler: InspectFungible<AccountIdFor<Self>> + MutateFungible<AccountIdFor<Self>>;
```

Comment:
Hmm, I kind of think it would be more intuitive to just call this associated type `Fungible`, what do you think?

Reply:
Hmm yeah, I agree!